ui/shared/utils/constants.js

```diff
@@ -1343,6 +1344,36 @@ export const PREDICTOR_FIELDS = [
   { field: 'mitotip', indicatorMap: MITOTIP_MAP },
   { field: 'hmtvar', thresholds: [undefined, undefined, 0.35, 0.35, undefined] },
 ]
+export const PRED_COLOR_MAP = ['green', 'olive', 'grey', 'yellow', 'red', '#8b0000']
```

Comment:
the name of this constant is okay for something used internally in a file, but if we are going to make it an importable constant we should probably name it something more decriptive and accurate, maybe `PREDICTION_COLOR_LIST` or `PREDICTION_COLORS`? Also, maybe it makes more sense not to export it at all and instead move all the helper functions that use it into this file, but thats up to you

Reply:
I moved the function using `PRED_COLOR_MAP` into here.  Seemed cleaner!